### PR TITLE
auth/info/ API: Return user information by user ID.

### PR DESCRIPTION
**Pull request by enkore**
This is the API that shall allow completion of qabel/qabel-block#42 /  implementation of qabel/qabel-block#30

It is orthogonal to the auth API, which returns user information via the users auth token: auth/info returns user information via the user ID.

### CODE REVIEW

File: qabel_provider/views.py [hunk 1]
```diff
@@ -81,6 +82,41 @@ def auth_resource(request, format=None):
...
 
+@api_view(('POST',))
+def info_resource(request, format=None):
+    """
+    Return user information by user ID.
+
+    Provides access to user information for other services like qabel-block.
+
+    :param request: rest request
+    :param format: ignored, because the resource never responds with a body
```
Comment (audax) [inline qabel_provider/views.py]: The ressource does respond with a body.

Comment (enkore) [inline qabel_provider/views.py]: Indeed. I will fix the docs here and the doc of auth_resource as well (copy-pasted).

**Comment by enkore** [comment]
Hm, looks like the GHPRB trigger didn't take. A build was started, but the parameter block was empty.

https://jenkins.prae.me/job/qabel-accounting-tests/90/parameters/

File: qabel_provider/test_rest.py [hunk 2]
```diff
@@ -127,9 +127,31 @@ def test_get_own_user(api_client, user):
         == loads(response.content)
 
 
-def test_auth_resource(external_api_client, user, token):
-    path = '/api/v0/auth/'
-    response = external_api_client.post(path, {'auth': 'Token {}'.format(token)})
+@pytest.fixture(params=['auth', 'info'])
+def call_resource_api(request, external_api_client, user, token):
```
Comment (audax) [inline qabel_provider/test_rest.py]: what about "call_auth_ressource" instead? "ressource_api" is a bit ambiguous.

File: qabel_provider/views.py [hunk 3]
```diff
@@ -81,6 +80,39 @@ def auth_resource(request, format=None):
...
 
 
+@api_view(('POST',))
+def info_resource(request, format=None):
+    """
+    Return user information by user ID.
+
+    Provides access to user information for other services like qabel-block.
+
+    :return: HttpResponseBadRequest|HttpResponse(status=204)|HttpResponse(status=403)
```
Comment (audax) [inline qabel_provider/views.py]: If you list the status, don't forget the 404

**Comment by enkore** [comment]
- Move "/api/v0/auth/" ($) to something else (inter-microservice API)
- Merge "info_resource" and "auth_resource"; either provide an auth token or a user ID

File: qabel_provider/views.py (hunk 3 above)
Comment (enkore) [inline qabel_provider/views.py]: I think I remove that part as well: this isn't the API documentation.

**Pull request merged**